en/techdocs/api/client/hoodie.md

```diff
@@ -29,253 +31,337 @@ This document describes the functionality of the hoodie base object.
-in the browser's local store. When signing in to an account,
-**hoodie.id()** will return the id of the new user.
-On signout, hoodie.id() gets reset.
+Each Hoodie user is assigned a random, unique, persistent ID on signup. While the user is logged in, **id()** will always return that same ID. When no user is signed in, it will return a different, non-persistent ID for the current anonymous user. Every time a user signs out, the ID for the anonymous user changes.
 
-##### Example
-<pre><code>hoodie.id(); // randomid123
+#### Example
+```javascript
+hoodie.id(); // "i67s6sm"
 hoodie.account.signIn(username, password)
   .done(function() {
-    hoodie.id(); // randomid456
+    hoodie.id(); // "vley9kp"
 });
-</code></pre>
-
+```
 
 <a id="on"></a>
-### on()
+### [on()](#on)
 **version:**      *> 0.2.0*
 
-*Bind an event handler to a certain event so you get informed in case it gets triggered.*
+**Binds a persistent event handler for specified events**
 
-<pre><code>hoodie.on('event', eventHandler); </code></pre>
+```javascript
+hoodie.on('event', eventHandler);
+```
 
-| option     | type   | description     | required |
-| ---------- |:------:|:---------------:|:--------:|
-| event        | String   | custom event identifier            | yes |
-| eventHandler | Function | Function handling triggered event. | yes |
+| option       | type     | description                        | required |
+|:------------ |:-------- |:---------------------------------- |:-------- |
+| event        | String   | Custom event identifier            | yes      |
+| eventHandler | Function | Function handling triggered event  | yes      |
 
-Hoodie informs you about several things happening. For instance every time a store object has been added, saved or removed. In order to catch those messages you can register a function for handling those events. Those functions are called **eventHandlers**. You can register also for events you created and triggered yourself. See [trigger](#trigger) for further information.
+Hoodie informs you about a lot of internal events, such as data being added, removed or changed, or a user signing in or out. These can be listened to with **on()**. You can also register listeners for events you created and triggered yourself. See [trigger](#trigger) more on this.
 
-##### Example
-<pre><code>hoodie.store.on('event', function(createdTodo) {
-  console.log('A todo has been added => ', createdTodo);
+#### A Note on the Event Identifier Format
+
+Hoodie store event identifiers always come in the same format:
+
+```javascript
+object-type:object-id:event-type
+```
+
+**Object-type** and **event-type** are mandatory. Omitting the **object-id** listens to events from all objects of a type. If you *do* specify an object id, you'll only receive events from that one specific object. If you want to listen for more than one event type at once, use the **change** type, which fires on *add*, *update* and *remove*.
```

Comment:
> +**Object-type** and **event-type** are mandatory

That's not correct. **Object-type** is optional. You can do `hoodie.store.on("update", handle)`.